transport/src/main/java/io/netty/channel/ManualIoEventLoop.java

```diff
@@ -48,6 +49,9 @@
  * {@link #waitAndRun()}} methods are called in a timely fashion.
  */
 public final class ManualIoEventLoop extends AbstractScheduledEventExecutor implements IoEventLoop {
+
+    private static final int
+            TICKER_FREQUENCY_MASK = Pow2.roundToPowerOfTwo(Integer.getInteger("io.netty.ticker.frequency", 64)) - 1;
```

Comment:
I think we have this kind of stuff in MathUtil.

Reply:
Let me change this - wdyt about adding this to #14556 as well